{{cookiecutter.bot_name}}/app/db/models.py

```diff
@@ -0,0 +1,59 @@
+"""Database models declarations."""
+
+from typing import Any
+
+from sqlalchemy import Column, Integer, String, update as sqlalchemy_update
+from sqlalchemy.future import select
+
+from app.db.sqlalchemy import Base, async_db_session
+
+
+class ModelAdminMixin:
+    """Mixin for CRUD operations for models."""
+
+    @classmethod
+    async def create(cls, **kwargs: Any) -> None:
+        """Create object."""
+        async_db_session.add(cls(**kwargs))  # type: ignore
+        await async_db_session.commit()
+
+    @classmethod
+    async def update(cls, id: int, **kwargs: Any) -> None:  # noqa: WPS125
+        """Update object by id."""
+        query = (
+            sqlalchemy_update(cls)
+            .where(cls.id == id)  # type: ignore
+            .values(**kwargs)
+            .execution_options(synchronize_session="fetch")
+        )
+
+        await async_db_session.execute(query)
+        await async_db_session.commit()
```

Comment:
Maybe use `begin` context manager to autocommit changes? You've done in file `app/db/sqlalchemy.py`.